sidecar/config/config.go

```diff
@@ -85,6 +85,8 @@ type Config struct {
 	LogstashServer string `yaml:"logstash_server"`
 
 	LogLevel string `yaml:"log_level"`
+
+	Debug string `yaml:"debug"`
```

Comment:
does it even make sense to have the debug flag in the YAML? can we get away without defining it here?